javascript/javascript1/week2/homework3.js

```diff
@@ -0,0 +1,37 @@
+// Weather wear
+
+const appropiateWearHelper = (temp) => {
+  console.log("--we only have support for jackets at the moment--");
+
+  const jacketDrawer = [
+    "Down Jacket",
+    "Insulated Jacket",
+    "Winter Jacket",
+    "Lightweight Jacket",
+    "Sports Jacket",
+    "Lightweight Hoodie",
+    "Fleece Jacket",
+    "Windbreaker",
+    "Leather Jacket",
+    "Parka",
+  ];
```

Comment:
Great that you're not afraid to use arrays!

I would say for this exercise that it wouldn't be necessary. Arrays are great for when you want to loop over an array/list of items. In this case it can make each console.log a bit harder to understand what it will be logging out as you would have to know the index and the word.